src/components/Comment-form.jsx

```diff
@@ -1,11 +1,69 @@
-import React from "react";
+import { useState } from "react";
 
-const CommentForm = () => {
-  return (
-    <form>
-      <label></label>
-    </form>
-  );
+const CommentForm = ({
+  comment,
+  setComments,
+  article_id,
+  postComment,
+  username,
+  setCommentCount,
+}) => {
+  const [newComment, setNewComment] = useState("Enter comment here...");
+  const [isValid, setIsValid] = useState(true);
+  if (comment) {
+    return (
+      <form
+        className="commentForm"
+        onSubmit={(e) => {
+          e.preventDefault();
```

Comment:
This logic could be handled with using some of the attributes available to the textarea element - required and placeholder spring to mind

https://www.w3schools.com/tags/tag_textarea.asp